tests/by-util/test_whoami.rs

```diff
@@ -50,3 +50,9 @@ fn test_normal_compare_env() {
         println!("test skipped:");
     }
 }
+
+#[test]
+#[cfg(windows)]
```

Comment:
Looks good! Thanks!

Maybe we could remove the `cfg` here and just name this test something like "test_succeeds_on_all_platforms". There might be other platforms where we skip the other test as well.